cli/crates/cli/tests/federation.rs

```diff
@@ -0,0 +1,46 @@
+#![allow(unused_crate_dependencies)]
+mod utils;
+
+use utils::environment::Environment;
+
+#[test]
+#[cfg(not(target_os = "windows"))]
```

Comment:
Just curious: why not windows?

Reply:
Not sure why, it's coming from the utilities to run typescript config in the tests